build.gradle

```diff
@@ -13,7 +13,19 @@ allprojects {
     }
     pluginManager.withPlugin('java') {
         java {
-            sourceCompatibility = JavaVersion.VERSION_11
+            toolchain {
+                languageVersion = JavaLanguageVersion.of(17)
+            }
+        }
+
+    }
```

Comment:
you can remove this one - it's not used in this build
(it is used in `integration-tests-spring` but that is actually a separate build)